server/tests/configuration.test.ts

```diff
@@ -1,35 +1,105 @@
-import { decodeConfiguration } from '../src/configuration'
+import { Octokit } from 'probot'
+import mls from 'multilines'
+import { getConfig } from '../src/configuration'
+import { Sources } from '../src/sources'
+import { getMockSources } from './__mock__/sources'
 
 describe('configuration:', () => {
-  test('errors on incorrect', () => {
-    const config = {
-      starters: [
-        {
-          name: 'starter',
-          path: '/templates/starter',
-        },
-        {
-          faulty: '',
+  describe('getConfig:', () => {
+    let sources: Sources = getMockSources()
+    let github: Octokit
+    let getContents: jest.Mock
```

Comment:
I prefer to keep jest mocks local to each test to make sure we don't accidentally override anything. What's your experience? 🙂